README.md

```diff
@@ -1757,18 +1757,12 @@ Log::addNotice('A notice error.');
 
 #### Get environment
 ```PHP
-$environment = Core::make('app')->environment(); //echo $environment;
+$environment = $app->environment(); //echo $environment;
 ```
 
 #### Clear site cache
 ```PHP
-Core::make('app')->clearCaches();
-//OR
-$this->app->clearCaches();
-//OR
-//use Concrete\Core\Support\Facade\Application;
-$app = Application::getFacadeApplication();
-$app->make('app')->clearCaches();
```

Comment:
`$app->make('app')` returns `$app` (that is `$app->make('app') === $app`), so it's useless.